server/service.go

```diff
@@ -29,6 +29,7 @@ var (
 	errServerAlreadyRunning = errors.New("server already running")
 )
 
+var slotDuration = time.Second * 12
```

Comment:
i'd call this `relayMonitorFaultPollInterval` and make it a 30sec for the beginning. probably should be configurable with env var (or cli flag, but that maybe later)